src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Json.cs

```diff
@@ -120,6 +122,34 @@ public override void CheckSemantics(TexlBinding binding, TexlNode[] args, DType[
 
                 if (nodeValue != null)
                 {
+                    foreach (var option in nodeValue)
+                    {
+                        switch (option)
+                        {
+                            case _ignoreBinaryDataEnumValue:
+                                ignoreBinaryData = true;
+                                break;
+                            case _ignoreUnsupportedTypesEnumValue:
+                                ignoreUnsupportedTypes = true;
+                                break;
+                            case _includeBinaryDataEnumValue:
+                                includeBinaryData = true;
```

Comment:
This is getting overwritten below at 154.  Why do we need to set this? Same for other flags.

Reply:
Good catch, the later code should be removed.